src/main/java/org/folio/rest/impl/NoteTypesImpl.java

```diff
@@ -112,7 +112,6 @@ public void deleteNoteTypesByTypeId(String typeId, String lang, Map<String, Stri
   @Override
   public void putNoteTypesByTypeId(String typeId, String lang, NoteType entity, Map<String, String> okapiHeaders,
       Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    removeNotStoredFields(entity);
```

Comment:
Deleted this method because fields that are not stored in database are declared as read only, so they will be null either way